### PR TITLE
Importing TPL lib for future imports

### PR DESCRIPTION
**Pull request by tjungbauer**
#### What is this PR About?
This PR is importing a tpl library which is required for future helm charts to be added.

#### How do we test this?
Inside README. 

cc: @redhat-cop/day-in-the-life

### CODE REVIEW

**Comment by tjungbauer** [comment]
How do we test this?
Inside README --> Inside README and example values you can find examples on how to use this library. It is basically used to stop code repetition inside the helm charts.

File: charts/tpl/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: tpl
+description: A library that shall collect certain templats to reuse them among different charts.
```
Comment (ckavili) [inline charts/tpl/Chart.yaml:3]:
```suggestion
description: A library that shall collect certain templates to reuse them among different charts.
```

small typo 😅

File: charts/tpl/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+  ![Version: 1.0.7](https://img.shields.io/badge/Version-1.0.7-informational?style=flat-square)
+
+ 
+
+  ## Description
+
+  A library that shall collect certain templats to reuse them among different charts.
```
Comment (ckavili) [inline charts/tpl/README.md:13]:
```suggestion
  A library that shall collect certain templates to reuse them among different charts.
```

**Comment by ckavili** [comment]
LGTM, thanks @tjungbauer 🚀

**Pull request merged**